pkg/config/config.go

```diff
@@ -75,14 +75,14 @@ type PrefixConfig struct {
 }
 
 // GetAdditionalColumns extracts AdditionalColumns defined for a given Resource
-func (c *Config) GetAdditionalColumns(resourceName string) []AdditionalColumnConfig {
+func (c *Config) GetAdditionalColumns(resourceName string) []*AdditionalColumnConfig {
 	if c == nil {
-		return []AdditionalColumnConfig{}
+		return []*AdditionalColumnConfig{}
```

Comment:
Just a nil return is sufficient
```suggestion
        return nil
```